src/main/java/net/dv8tion/jda/internal/handle/GuildRoleUpdateHandler.java

```diff
@@ -74,7 +74,7 @@ protected Long handleInternally(DataObject content)
             role.setName(name);
             getJDA().handleEvent(
                     new RoleUpdateNameEvent(
-                            getJDA(), responseNumber,
+                            getJDA(), responseNumber, getJDA().isEventPassthrough() ? content : null,
```

Comment:
It makes more sense to just handle this inside the event constructor, instead of repeating the same check everywhere